spec/inquisition/outputter/xlsx/builder_spec.rb

```diff
@@ -0,0 +1,24 @@
+RSpec.describe Inquisition::Outputter::XLSX::Builder do
+
+  after { FileUtils.rm_rf(Inquisition::Configuration.instance.output_path) }
```

Comment:
It would be better to move this hook where it's really needed since it's used only for one test.